### PR TITLE
Require authentication via api key or client id

### PR DESCRIPTION
**Pull request by Caerostris**
As of June 22nd 2016, the Google Geocoding API requires authentication via api key or clientId/privateKey pair

### CODE REVIEW

File: src/geocoder.js [hunk 1]
```diff
@@ -11,6 +11,24 @@ const defaults = {
   apiKey: null
 };
 
+function validateOptions(options) {
```
Comment (ro-ka) [inline src/geocoder.js:19]: Add a DocBlock comment.

**Comment by ro-ka** [comment]
🏁

**Comment by Caerostris** [comment]
Plus, I forgot to remove the varying queries per second, which is now obsolete.

**Comment by TheAmazingPT** [comment]
@ro-ka, @Caerostris  Please merge, i need these error output improvements badly for another project!! :D

**Comment by ro-ka** [comment]
Released as `1.3.0`. Have fun badging!  😺

**Pull request merged**